packages/webapp/src/components/Map/SelectionHandler/index.jsx

```diff
@@ -1,24 +1,102 @@
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import clsx from 'clsx';
 import { areaImgDict, lineImgDict, pointImgDict } from '../LocationMapping';
-import { containsCrops, longPress } from '../../../containers/Map/constants';
+import { ReactComponent as ShowMore } from '../../../assets/images/map/arrowDown.svg';
+import { containsCrops } from '../../../containers/Map/constants';
 import { makeStyles } from '@material-ui/core/styles';
 import { colors } from '../../../assets/theme';
-import PurePreviewPopup from '../PreviewPopup';
+import CompactPreview from '../PreviewPopup/CompactPreview';
 import { SENSOR } from '../../../containers/SensorReadings/constants';
 import { isTouchDevice } from '../../../util/device';
+import { useSelector } from 'react-redux';
+import { sensorReadingTypesByMultipleLocations } from '../../../containers/sensorReadingTypesSlice';
+import { TEMPERATURE, SOIL_WATER_POTENTIAL } from '../../../containers/SensorReadings/constants';
 
 const useStyles = makeStyles((theme) => ({
   container: {
-    left: 100,
-    '&:hover': {
-      backgroundColor: colors.green100,
-    },
+    position: 'relative',
+  },
+  arrow: {
+    position: 'absolute',
+    width: 15,
+    height: 15,
+    borderColor: 'transparent',
+    borderRightColor: 'transparent',
+    borderStyle: 'solid',
+    left: '45%',
+    marginTop: -20,
+    borderWidth: '10px 10px 10px 10px',
+    borderBottomColor: 'white',
+  },
+  tooltip: {
+    position: 'absolute',
+    left: -140,
+    top: 30,
+    pointerEvents: 'initial',
     backgroundColor: 'white',
-    marginBottom: '5px',
-    marginLeft: 'auto',
-    marginRight: 'auto',
-    maxWidth: 160,
+    boxShadow: '2px 6px 12px rgba(102, 115, 138, 0.2)',
+    padding: 0,
+    borderRadius: '4px',
+    width: '290px',
+    marginTop: 5,
+    userSelect: 'none',
+    color: colors.grey900,
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontFamily: 'Open Sans, SansSerif, serif',
+  },
+  tooltipWithSoilWaterPotential: {
+    width: '290px',
+    left: -140,
+  },
+  itemHeader: {
+    padding: '0 8px 0 8px',
+  },
+  title: {
+    padding: '8px 0 0 0',
+    fontWeight: 'bold',
+    borderBottom: '1px solid',
+    borderBottomColor: 'var(--grey400)',
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  itemIcon: {
+    padding: '0 8px',
+    height: 24,
+  },
+  sensorIcon: {
+    '& svg': {
+      width: 24,
+      height: 24,
+      transform: 'scale(1.25)',
+    },
+  },
+  itemName: {
+    flexGrow: 1,
+    padding: '0 8px',
+    lineBreak: 'auto',
+  },
+  sensorBody: {
+    height: 'auto',
+    transition: 'all 1s',
+    overflow: 'hidden',
+  },
+  bodyClosed: {
+    maxHeight: 0,
+  },
+  bodyOpen: {
+    maxHeight: '65px',
+    paddingBottom: '8px',
+  },
+  sensorArrow: {
+    display: 'flex',
+    flexDirection: 'row',
+    alginSelf: 'center',
+    transition: 'transform 1s',
+  },
+  sensorArrowUp: {
+    transform: 'rotate(-180deg)',
```

Comment:
Nice animation :)
(I'm not sure what Carolina will say, but personally I prefer faster 😆)

Reply:
I did sent a video to Caro, she approved it as it was, so I guess I will leave it that way.